### PR TITLE
Add wrap option to Tooltips

### PR DESCRIPTION
**Pull request by jerseygooner**

### CODE REVIEW

**Comment by ajkitson** [comment]
Can we just give `.rui-tooltip` a `max-width` for 300px? Then it'll wrap at 300px for anything that wide and will just be smaller for anything else and we wouldn't need the extra attribute on the directive.

**Comment by jerseygooner** [comment]
I tried this, but it doesn't work that way for a floating tooltip. It auto-wraps to the largest word instead. I tried this several different ways and found an example where javascript may be the only way to do this: 
http://stackoverflow.com/questions/5683364/how-to-make-pure-css-floating-tooltips-absolutely-positioned-span-dynamically

**Comment by ajkitson** [comment]
Ah, I see--that is tough then. If we can make the JS version work, that seems to be best. That way it'll work for short helptext and long tooltips.

File: dst/redox-ui-components.js [hunk 1]
```diff
@@ -433,21 +433,45 @@ app.directive('ruiToggle', ['$compile' ,function ($compile) {
...
+        tooltip.addClass('rui-tooltip-show');
+
+        //find the container of our element so that we can position our tooltip
+        var containerPosition = e.target.getBoundingClientRect(),
+          offset = {},
+          tooltipHeight = tooltip.outerHeight(),
+          topOffset = containerPosition.top + window.pageYOffset; //account for scrolling of the window
+
+        offset.top = topOffset - tooltipHeight - 10; //-10 for the down arrow
+        offset.left = containerPosition.left + (containerPosition.width / 2) - 31;
```
Comment (ajkitson) [inline dst/redox-ui-components.js]: Why -31?

Comment (jerseygooner) [inline dst/redox-ui-components.js]: This is the offset for the arrow - adding a comment as I did for the line above.

**Comment by ajkitson** [comment]
Looks good! :fire:

**Pull request merged**